### ISSUE TITLE
Unimplemented names

### ISSUE THREAD

**Issue by targos** [cheminfo/biologic-converter#36]
- [ ] GEIS: https://github.com/zakodium-oss/analysis-dataset/tree/main/docs/data/format/biologic/geis
- [ ] MB: https://github.com/zakodium-oss/analysis-dataset/tree/main/docs/data/format/biologic/mb
- [ ] PEIS: https://github.com/zakodium-oss/analysis-dataset/tree/main/docs/data/format/biologic/peis and https://github.com/zakodium-oss/analysis-dataset/blob/main/docs/data/format/biologic/vsp_peis_with.mpr

**Comment by ghost** [comment]
Yes, I am aware, it is just that it gets more complicated for some of those. I will take a look so that if I can't do it at least I can write a brief for the next person trying. Apart from that, I am weary about writing code too inspired in code written by others (in this particular case.)

**Comment by targos** [comment]
> I am weary about writing code too inspired in code written by others (in this particular case.)

I don't know what you have in mind, but unless the "code written by others" you are talking about is secret and you are not supposed to have access to it, this shouldn't be a problem.
If I understand correctly, you read the code to get information about the file format, not to copy an algorithm.

**Comment by ghost** [comment]
Interesting. Yes I guess I am just a bit scared and lack experience, I have been explained this already.

**Comment by ghost** [comment]
Basic GEIS and PEIS is being added in the PR referenced above @targos. Still needs a bit of work. The parameter names are now as they appear in the text file for convenience.
